zipkin-server/src/main/java/zipkin2/server/internal/ui/ZipkinUiConfiguration.java

```diff
@@ -36,12 +37,10 @@
 import io.netty.handler.codec.http.cookie.ServerCookieDecoder;
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.UncheckedIOException;
+import java.io.InputStreamReader;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
-import org.jsoup.Jsoup;
```

Comment:
jsoup was a large dependency, and we are only replacing a href tag in staticly defined index.html under our control. regex will do the trick.